prog/mutation.go

```diff
@@ -258,15 +258,18 @@ func mutateInt(r *randGen, s *state, arg Arg) (calls []*Call, retry, preserve bo
 	if r.bin() {
 		return regenerate(r, s, arg)
 	}
+	argSize := uint64(math.Min(float64(arg.Size()), 8))
```

Comment:
No floats please. Float point creates lots of problems. E.g. 8 may not be equal to 8 because it's actually 7.999999999999999999999999999. Object size calculations must not involve floats at all.